pages/api/recipes/index.js

```diff
@@ -0,0 +1,10 @@
+import { getAllEvents } from "../../../data/db";
+
+export default async function handler(req, res) {
+  if (req.method === "GET") {
+    const events = await getAllEvents();
+    res.status(200).json(events);
+  } else {
+    res.status(405).setHeader("Allow", ["GET"]).send();
+  }
+}
```

Comment:
See my comment above

```suggestion
import { getAllDishes } from "../../../data/db";

export default async function handler(req, res) {
  if (req.method === "GET") {
    const dishes = await getAllDishes();
    res.status(200).json(dishes);
  } else {
    res.status(405).setHeader("Allow", ["GET"]).send();
  }
}
```